samples/boards/nrf91/nrfx/README.rst

```diff
@@ -0,0 +1,45 @@
+.. _nrf91_nrfx:
+
+nRF91 nrfx example
+##################
+
+Overview
+********
+
+This sample demonstrates the usage of nrfx library in Zephyr.
+GPIOTE and DPPI are used as examples of nrfx drivers.
+
+The code shows how to use nrfx directly by connecting the GPIOTE interrupt to Zephyr
+in conjunction with DPPI subsystem.
+Button 1 is configured to create an event when pushed.
+This calls the ``button_handler()`` callback and triggers the LED 1 clear
+task. LED is then turned off. Every 500ms a set event for the LED pin is triggered.
+This causes the LED to turn on.
```

Comment:
This is a bit strange behavior. When the user presses the button, the LED may be turned off for 500 ms, but it can also be turned off for a very short time, even unnoticeable to the user. I'm afraid this may be confusing.
I'd rather refactor the code a bit, so that the LED is toggled via DPPI on each button press. Actually, after configuring DPPI, you could just exit `main`. Both the interrupt handler and the DPPI connection will still work. This will show better what DPPI can provide.

Reply:
Changed to OUT event, now the LED is toggled on button press. Deleted the infinite loop.